swan-lake/development-tutorials/ballerina-persist/supported-data-stores.md

```diff
@@ -246,8 +247,88 @@ Follow the steps below to obtain API tokens for Google Sheets.
 1. Get the `clientID` and `ClientSecret` by following the [guidelines](https://developers.google.com/identity/protocols/oauth2).
 2. Follow the steps below to add https://developers.google.com/oauthplayground if you want to use OAuth 2.0 playground to receive the authorization code, access token, and refresh token.
 
-   1. Go to the OAuth 2.0 playground and click **Settings** on the top right-hand side.
-   2. Select **Use your own OAuth credentials** and provide the client ID and client secret obtained.
-   3. For step 1 under the  `Select & authorize APIs` section, select the required scopes for the API you are going to use.
-   4. Proceed with the necessary Google account validation steps.
-   5. Click **Exchange authorization code for tokens** to obtain the refresh and access tokens.
\ No newline at end of file
+    1. Go to the OAuth 2.0 playground and click **Settings** on the top right-hand side. 
+    2. Select **Use your own OAuth credentials** and provide the client ID and client secret obtained. 
+    3. For step 1 under the  `Select & authorize APIs` section, select the required scopes for the API you are going to use. 
+    4. Proceed with the necessary Google account validation steps. 
+    5. Click **Exchange authorization code for tokens** to obtain the refresh and access tokens.
+
+## Redis [Experimental]
+
+The Redis data store is a key-value-based inmemory datastore useful for storing and retrieving unstructured data quickly. It is not the default data store for `bal persist`. Therefore, you need to explicitly specify the data store when initializing `bal persist` in your application as follows.
+
+```shell
+$ bal persist init
+```
+
+### Supported Ballerina types
+
+The Redis data store supports the following Ballerina types. As we have utilized `hash` and `set` as Redis data types to store objects, the following data types will be converted to `string`.
+
+- int
+- float
+- decimal
+- string
+- time:Date
+- time:TimeOfDay
+- time:Utc
+- time:Civil
+- enum
+
+### Configuration
+
+You need to set values for the following basic configuration parameters in the `Config.toml` file in your project to use the Redis data store.
```

Comment:
while this says `following basic configuration parameters` it only lists one parameter in the table. If it is only a single parameter , do we really need a table below?

Reply:
There are 4 parameters in total,

- `connection`
- `connectionPooling`
- `isClusterConnection`
- `secureSocket`

Default values have been set for `connection`, `connectionPooling`, and `isClusterConnection` in the Redis connector itself, and the parameter `secureSocket` is an optional parameter. The reason we're giving the `connection` parameter in `Config.toml` is that it's easy for the user to replace the connection URI in case of a cloud-hosted database instance. Users may need to use values other than the default values for parameters `connectionPooling` and `isClusterConnectionj` as well. In that case, having a table is more cleaner in my opinion. Any suggestions to solve this?

Reply:
I think I've misunderstood the question. This is regarding the table in the markdown file, right? Yes, in that case, we don't need a table here. I'll make the necessary changes.

Reply:
This has been fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/8838/commits/4612960007eaacc7c43ef86420f1bddaf4b13c74